server/build/babel/plugins/handle-import.js

```diff
@@ -9,7 +9,7 @@ const TYPE_IMPORT = 'Import'
 
 const buildImport = (args) => (template(`
   (
-    typeof window === 'undefined' ?
+    typeof require.resolveWeak !== 'function' ?
```

Comment:
Could you add a comment where why we use this rather a simple `window` check.
It's better for someone who's reading the code.